### ISSUE TITLE
Time Zone Set error

### ISSUE THREAD

**Issue by pilondav** [dkerr64/homebridge-yolink#95]
**Describe The Bug:**
I'm getting an error in the syslog when I use the "Sync With Mobile" feature in the YoLink app for iOS for the YoLink YS5002-UC water valve.

**To Reproduce:**
Open YoLink app for iOS. Select water valve tile. Select Details hamburger menu. Click "Sync With Mobile" option. Click "Confirm." Notice log entry below.

**Expected behavior:**
No error in logs.

**Logs:**
[4/6/2024, 11:52:18 AM] [YoLink] MQTT: Manipulator.setTimeZone for device YS5002-UC (d88b4c010007243c) Main Water Valve not supported.
Please report all bugs at https://github.com/dkerr64/homebridge-yolink/issues
{
  "event": "Manipulator.setTimeZone",
  "time": 1712418738029,
  "msgid": "1712418738028",
  "data": {
    "tz": -4
  },
  "deviceId": "d88b4c010007243c"
}```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
{
    "bridge": {
        "name": "Homebridge 94EE",
        "username": "xxx",
        "port": 51096,
        "pin": "xxx",
        "advertiser": "bonjour-hap"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "name": "YoLink",
            "platform": "YoLink",
            "tokenURL": "https://api.yosmart.com/open/yolink/token",
            "apiURL": "https://api.yosmart.com/open/yolink/v2/api",
            "mqttPort": 8003,
            "userAccessId": "xxx",
            "secretKey": "xxx",
            "refreshAfter": 14500,
            "verboseLog": false,
            "liteLog": true,
            "allDevices": true,
            "excludeTypes": [
                "Hub",
                "SpeakerHub"
            ],
            "deviceTemperatures": true,
            "powerFailureSensorAs": "Contact",
            "enableExperimental": false,
            "garageDoors": [
                {
                    "controller": "xxx",
                    "sensor": "xxx",
                    "timeout": 45
                }
            ],
            "_bridge": {
                "username": "xxx",
                "port": 51262
            }
        }
    ]
}```

**Screenshots:**
<!-- If applicable, add screenshots to help explain your problem. -->

**Environment:**

* **Plugin Version**:  1.5.5
* **Homebridge Version**: 1.7.0
* **Node.js Version**: 20.9.0
* **NPM Version**: unsure, assume current version
* **Operating System**: Synology

<!-- Click the "Preview" tab before you submit to ensure the formatting is correct. -->

**Comment by dkerr64** [comment]
Thank you for reporting this.  I was not aware that we might get a Manipulator.setTimeZone event.  I would just ignore it anyway, so you can continue using the plugin without problems.  Next time I push an update I'll suppress this message.

Thanks again for reporting it.

**Comment by pilondav** [comment]
Ok, thanks! It doesn’t seem to be causing any trouble on my end.DaveClear eyes, full hearts, can't lose.On Apr 7, 2024, at 5:06 PM, David Kerr ***@***.***> wrote:﻿
Thank you for reporting this.  I was not aware that we might get a Manipulator.setTimeZone event.  I would just ignore it anyway, so you can continue using the plugin without problems.  Next time I push an update I'll suppress this message.
Thanks again for reporting it.

**Comment by dkerr64** [comment]
Fixed in release 1.5.6
